pkg/views/workspace/selection/repository.go

```diff
@@ -14,19 +14,32 @@ import (
 	tea "github.com/charmbracelet/bubbletea"
 )
 
-func selectRepositoryPrompt(repositories []apiclient.GitRepository, index int, choiceChan chan<- string) {
+func selectRepositoryPrompt(repositories []apiclient.GitRepository, index int, choiceChan chan<- string, providerId string, selectedReposGitProvider map[string]bool, selectedRepos map[string]bool) {
```

Comment:
![Screenshot 2024-07-08 at 09 48 49](https://github.com/daytonaio/daytona/assets/26512078/8214fba5-bbe3-46d7-ae24-419a613c0fc4)

If I select the first repo in the list, it will be focused the second time I enter the picker.
Please make it so the first focused repo is the first one that is not disabled.

I get that I can not press enter here, but it is still a bit confusing since when I move focus, I can no longer return it to this repo.

Reply:
@Tpuljak  you mean just keeping the first option non-disabled always? 
Anyways a side question, Do you have any idea how to move the cursor to a non-disabled option on every first render of the view? I tried writing some logic like below in Init() method nothing worked out sadly. 😞 

```
func (m model[T]) Init() tea.Cmd {
	// Ensure the cursor starts from the first non-disabled item
	for i := 0; i < len(m.list.Items()); i++ {
		item := m.list.Items()[i].(item[T])
		if !item.disabled {
			m.list.Select(i)
			break
		}
		OR move cursordown() logic  nothing works
	}
```

Reply:
also the reason why i blocked 'enter' keypress for disabled options..

Reply:
> @Tpuljak you mean just keeping the first option non-disabled always?

No, it should be disabled.

> Anyways a side question, Do you have any idea how to move the cursor to a non-disabled option on every first render of the view?

Whenever you update a property of the model that should be reflected in the TUI, you need to call `m.Update` so it gets rerendered. Try with that.